ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -357,8 +358,9 @@ export class AptosClient {
    * Generates and submits a transaction to the transaction simulation
    * endpoint. For this we generate a transaction with a fake signature.
    *
-   * @param accountFrom The account that will be used to send the transaction
-   * for simulation.
+   * @param accountOrPubkey The sender or sender's public key. When private key is available, `AptosAccount` instance
+   * can be used to send the transaction for simulation. If private key is not available, sender's public key can be
+   * used to send the transaction for simulation.
```

Comment:
Maybe:

If an `AptosAccount` is given, it will use the private key to sign the transaction for simulation. If not, it will generate signed bytes for you and use the provided public key. 

Or something like that.